### PR TITLE
DEP PHP Support in CMS5

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/silverstripe-framework/issues/10402

### CODE REVIEW

**Comment by sabina-talipova** [comment]
https://github.com/silverstripe/silverstripe-userforms/pull/1182 should be merged first

File: composer.json [hunk 1]
```diff
@@ -19,15 +19,15 @@
...
     "prefer-stable": true,
     "require": {
-        "php": "^7.4 || ^8.0",
-        "silverstripe/framework": "^4.10",
-        "dnadesign/silverstripe-elemental": "^4.0",
-        "silverstripe/userforms": "^5.0",
-        "silverstripe/vendor-plugin": "^1.0"
+        "php": "^8.1",
+        "silverstripe/framework": "^5",
+        "dnadesign/silverstripe-elemental": "5.0.0-alpha2",
```
Comment (maxime-rainville) [inline composer.json]:
```suggestion
        "dnadesign/silverstripe-elemental": "^5.0.0-alpha2",
```

Comment (maxime-rainville) [reply composer.json]: That will allow the installation of later 5.x version.

**Review by maxime-rainville — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (sabina-talipova) [reply composer.json]: DONE

**Comment by sabina-talipova** [comment]
Broken Behat tests are related to the issue https://github.com/silverstripeltd/product-issues/issues/655

**Review by maxime-rainville — APPROVED** [review]
LGTM

**Pull request merged**